src/main/java/lotto/domain/LottoFactory.java

```diff
@@ -0,0 +1,33 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class LottoFactory {
+
+    private static final List<Integer> DEFAULT_NUMBERS = IntStream
+            .rangeClosed(1, 45)
+            .boxed()
+            .collect(Collectors.toList());
+
+    private static final int LOTTO_SIZE = 6;
+
+    public static List<Integer> generateLotto() {
+        List<Integer> numbers = new ArrayList<>(DEFAULT_NUMBERS);
+        Collections.shuffle(numbers);
```

Comment:
```suggestion
        DEFAULT_NUMBERS.shuffle(numbers);
```
굳이 또 다른 ArrayList로 생성할 필요 있을까?

Reply:
DEFAULT_NUMBERS는 공용으로 사용하는거라서 새로 생성해서 분류했는데,
생각해보니 DEFAULT_NUMBERS의 역할은 서로 다른 섞여있는 번호를 가져오는 역할이기 때문에 생성할 필요는 없을거 같다는 생각이 드네요! 감사합니다.